src/app/shared/mastodon/mastodon.component.html

```diff
@@ -0,0 +1,103 @@
+<!--
+  ~    Copyright 2023 OICR
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<div class="mt-timeline">
+  <mat-card class="h-100">
+    <mat-card-header class="mt-header">
+      <mat-card-title class="mt-header-text" class="ml-2"
+        >Mastodon Posts from <a class="mt-header-link" [href]="mastodonDockstoreLink">@dockstore</a></mat-card-title
```

Comment:
Nit: all the other links in this component open a new tab but this one doesn't. Should make it open in a new tab so the behaviour is consistent with the other links